ui/shared/status/StatusWalletColorSelect.qml

```diff
@@ -0,0 +1,49 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import "../../imports"
+import "../../shared"
+import "../../shared/status"
+
+Item {
+    id: control
+    property string selectedColor
+    property string label: qsTr("Account color")
+    property var model
+    height: label.height + colors.height + 8
```

Comment:
`childrenRect.height` should work since you don't use `anchors.fill` or `veriticalCenter`, etc.